services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_reports/diagnostics/postTestTable.tsx

```diff
@@ -0,0 +1,194 @@
+import * as React from 'react'
+
+import {
+  correctImage,
+  baseDiagnosticImageSrc,
+} from './shared'
+
+import {
+  helpIcon,
+  Tooltip,
+  smallWhiteCheckIcon,
+} from '../../../../../Shared/index'
+import useWindowSize from '../../../../../Shared/hooks/useWindowSize'
+
+interface StickyTableStyle {
+  position: string,
+  top: number,
+  left: number,
+  right: number,
+  zIndex: number,
+}
+
+const ellipsesIcon = <img alt="Open menu icon" src={`${baseDiagnosticImageSrc}/ellipses_icon.svg`} />
+
+const PostTestAssignmentTable = ({showPostTestAssignmentColumn, students, studentsWhoCompletedDiagnostic, postTestSelections, setPostTestSelections, previouslyAssignedPostTestStudentIds, studentsWhoCompletedAssignedRecommendations, postDiagnosticUnitTemplateId, }) => {
+
+  const size = useWindowSize();
+  const [isSticky, setIsSticky] = React.useState(false);
+  const [stickyTableStyle, setStickyTableStyle] = React.useState<StickyTableStyle>({
+    position: "fixed",
+    top: 0,
+    left: 0,
+    right: 0,
+    zIndex: 5
+  })
+  const [stickyTableRightOffset, setStickyTableRightOffset] = React.useState(0)
+  const recommendationsTableRef = React.useRef(null);
+  const handleScroll = React.useCallback(({ top, bottom, left, right, }) => {
+    if (top <= 0 && bottom > 92) {
+      setStickyTableStyle(oldStickyTableStyle => ({ ...oldStickyTableStyle, left: left }))
+      setStickyTableRightOffset(782)
```

Comment:
`782` here should be a constant so we both have a little more context for where it's coming from (the name) and also can change it more easily.